Core/src/org/sleuthkit/autopsy/corecomponents/DataResultTopComponent.java

```diff
@@ -18,18 +18,23 @@
  */
 package org.sleuthkit.autopsy.corecomponents;
 
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import org.sleuthkit.autopsy.corecomponentinterfaces.DataResult;
 import java.util.logging.Level;
+import javax.swing.JComponent;
+import javax.swing.KeyStroke;
 import org.openide.explorer.ExplorerManager;
 import org.openide.explorer.ExplorerUtils;
 import org.openide.util.NbBundle;
 import org.openide.windows.TopComponent;
 import org.openide.nodes.Node;
 import org.openide.windows.Mode;
 import org.openide.windows.WindowManager;
+import org.sleuthkit.autopsy.actions.BookmarkFileAction;
```

Comment:
This does not compile. BookmarkFileAction is not public. If this is the only place the class is needed, we would leave it as package access and put it in this package rather than add a new public class.